cdp_scrapers/types.py

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from typing import NamedTuple, Optional
+from typing import Dict, NamedTuple, Optional
+from cdp_backend.pipeline.ingestion_models import (
+    Body,
+    Person,
+    Seat,
+)
 
 ###############################################################################
 
 
 class ContentURIs(NamedTuple):
     video_uri: Optional[str]
     caption_uri: Optional[str] = None
+
+
+class ScraperStaticData(NamedTuple):
+    seats: Dict[str, Seat] = None
+    primary_bodies: Dict[str, Body] = None
+    persons: Dict[str, Person] = None
```

Comment:
Not now, but in the future, I recommend we switch this over to a `dataclass_json` so that you can call `ScaperStaticData.from_json` to read and parse and validate in a single go.